less/mixins.less

```diff
@@ -895,15 +895,18 @@
 // Example usage: change the default blue border and shadow to white for better
 // contrast against a dark gray background.
 
-.form-control-focus(@color: @input-border-focus) {
-  @color-rgba: rgba(red(@color), green(@color), blue(@color), .6);
+.form-control-focus(@color: @brand-primary) {
+  @color-rgba: rgba(red(@color), green(@color), blue(@color), 1);
   &:focus {
-    border-color: @color;
-    outline: 0;
-    .box-shadow(~"inset 0 1px 1px rgba(0,0,0,.075), 0 0 8px @{color-rgba}");
+    border: 1px solid @color-rgba;
+    outline: none;
+    -webkit-box-shadow: none;
+    -moz-box-shadow: none;
```

Comment:
You can probably drop this.  Firefox dropped the prefix from version 4 upwards.  You haven't included it in the `css` version anyway.